src/utils.js

```diff
@@ -123,3 +123,13 @@ export function enforceMaxBodiesSize(ctx, tx) {
   ctx.totalBodyLength += len;
   return enforced;
 }
+
+export function getExtraMongoConfig() {
```

Comment:
Maybe rename this function to `getMongoOptions()` as this is only additional config options